tools/SetupFlow/DevHome.SetupFlow/Models/CloneRepoTask.cs

```diff
@@ -31,6 +31,7 @@ namespace DevHome.SetupFlow.Models;
 /// </summary>
 public partial class CloneRepoTask : ObservableObject, ISetupTask
 {
+

```

Comment:
Will build complain about this blank line?

Reply:
Yes.  Removed.